src/praisonai-code/praisonai_code/cli/commands/auth.py

```
@@ -168,14 +193,18 @@ def auth_login(
-                return
+            from ..configuration.oauth import provider_supports_oauth, provider_requires_client_id
+            if provider_supports_oauth(provider, oauth_overrides):
+                # Don't silently start an OAuth flow that will immediately fail
+                # for lack of a client id; fall through to API-key login unless
+                # the user explicitly supplied one.
+                if not provider_requires_client_id(provider, oauth_overrides):
+                    _run_oauth_login(provider, output, base_url, model, no_browser, oauth_overrides)
+                    return
         except Exception:
             # Fall through to API-key login on any OAuth detection error.
             pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)